beginner_source/quickstart/quickstart_tutorial.py

```diff
@@ -0,0 +1,253 @@
+        pred = model(X)
+        loss = cost(pred, Y)
+        loss.backward()
+        optimizer.step()
+    
+        if batch % 100 == 0:
+            loss, current = loss.item(), batch * len(X)
+            print(f'loss: {loss:>7f}  [{current:>5d}/{size:>5d}]')
+
+
+# Create the validation/test function
+
+def test(dataloader, model):
+    size = len(dataloader.dataset)
+    model.eval()
+    test_loss, correct = 0, 0
+
+    with torch.no_grad():
+        for batch, (X, Y) in enumerate(dataloader):
+            X, Y = X.to(device), Y.to(device)
+            pred = model(X)
+
+            test_loss += cost(pred, Y).item()
+            correct += (pred.argmax(1) == Y.argmax(1)).type(torch.float).sum().item()
+
+    test_loss /= size
+    correct /= size
+
+    print(f'\nTest Error:\nacc: {(100*correct):>0.1f}%, avg loss: {test_loss:>8f}\n')
+
+
+# Call the train and test function in a training loop with the number of epochs indicated.
+
+epochs = 5
+
+for t in range(epochs):
+    print(f'Epoch {t+1}\n-------------------------------')
+    train(train_dataloader, model, cost, optimizer)
+    test(test_dataloader, model)
+print('Done!')
+
+######################################################################
+# Saving Models
+# -------------
+# 
+# PyTorch has different ways you can save your model. One way is to serialize the internal model state to a file. Another would be to use the built-in `ONNX <https://github.com/onnx/tutorials>`_ support.
+# 
+
+torch.save(model.state_dict(), 'model.pth')
+print('Saved PyTorch Model to model.pth')
+
+# Save to ONNX, create dummy variable to traverse graph
+
+x = torch.randint(255, (1, 28*28), dtype=torch.float).to(device) / 255
+onnx.export(model, x, 'model.onnx')
+print('Saved onnx model to model.onnx')
+
+######################################################################
+# Loading Models
+# ----------------------------
+# 
+# Once a model has been serialized the process for loading the
+# parameters includes re-creating the model shape and then loading
+# the state dictionary. Once loaded the model can be used for either
+# retraining or inference purposes (in this example it is used for
+# inference). Check out more details on `saving, loading and running models with PyTorch <saveloadrun_tutorial.html>`_
+#
+
+loaded_model = NeuralNetwork()
+
+loaded_model.load_state_dict(torch.load('model.pth'))
+loaded_model.eval()
+
+# inference
+x, y = test_data[0][0], test_data[0][1]
+with torch.no_grad():
+    pred = loaded_model(x)
+    predicted, actual = classes[pred[0].argmax(0)], classes[y.argmax(0)]
+    print(f'Predicted: "{predicted}", Actual: "{actual}"')
+
+
+#############################################################
+#
+# Looking for more resources? Check out the other tutorials on the `tutorials home page <https://pytorch.org/tutorials/>`_.
+#
+
+##################################################################
+#
+# *Authors: Seth Juarez, Cassie Breviu, Dmitry Soshnikov, Ari Bornstein*
```

Comment:
Author links

```suggestion
# *Authors: Seth Juarez, Cassie Breviu, Dmitry Soshnikov, Ari Bornstein, `Suraj Subramanian <https://github.com/suraj813>`__*
```

Reply:
There is a conflict here. I will commit this change.

Reply:
Added!